### PR TITLE
Adding more samples

### PR DESCRIPTION
**Pull request by Tapchicoma**
#30: Added more samples that will able easier to test changes.

### CODE REVIEW

File: samples/android-app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,36 @@
+apply plugin: 'android'
```
Comment (JLLeitschuh) [inline samples/android-app/build.gradle]: Can't you use the Gradle Script Kotlin version of the DSL/API?

File: samples/android-app/proguard-rules.pro [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```
Comment (JLLeitschuh) [inline samples/android-app/proguard-rules.pro]: Is this the "normal" proguard directory? Don't project config things normally go in a `config` file?
Eg: `config/checkstyle`, `config/proguard`?

File: samples/android-app/src/androidTest/java/by/egorr/myapplication/ExampleInstrumentedTest.kt [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package by.egorr.myapplication
```
Comment (JLLeitschuh) [inline samples/android-app/src/androidTest/java/by/egorr/myapplication/ExampleInstrumentedTest.kt]: `eggor`? `by`?

File: samples/android-app/src/main/java/org/jlleitschuh/gradle/ktlint/myapplication/ItemListActivity.kt [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+
+import by.egorr.myapplication.dummy.DummyContent
+import kotlinx.android.synthetic.main.activity_item_list.*
+import kotlinx.android.synthetic.main.item_list_content.view.*
+
+import kotlinx.android.synthetic.main.item_list.*
```
Comment (JLLeitschuh) [inline samples/android-app/src/main/java/org/jlleitschuh/gradle/ktlint/myapplication/ItemListActivity.kt]: No star imports?

File: settings.gradle [hunk 5]
```diff
@@ -0,0 +1,12 @@
+rootProject.name = "ktlint-gradle-samples"
+
+include(':kotlin-ks')
+project(':kotlin-ks').projectDir = new File('./samples/kotlin-ks')
```
Comment (JLLeitschuh) [inline settings.gradle]: Can't you just use `project(':kotlin-ks').projectDir = file('./samples/kotlin-ks')`?

File: settings.gradle [hunk 6]
```diff
@@ -0,0 +1,12 @@
+rootProject.name = "ktlint-gradle-samples"
```
Comment (JLLeitschuh) [inline settings.gradle]: You can now write this file using Kotlin if you want to `settings.gradle.kts`

**Comment by JLLeitschuh** [comment]
Can you add to travis to run the samples as well?

**Comment by Tapchicoma** [comment]
Current setup runs `ktlintCheck` for all samples. I don't see a point to run samples themselfs - it is not a goal of this project.

**Comment by JLLeitschuh** [comment]
> Current setup runs ktlintCheck for all samples.

Okay, that's what I want. Thanks!

File: samples/android-app/build.gradle (hunk 1 above)
Comment (Tapchicoma) [reply samples/android-app/build.gradle]: Changed in 471d810

File: samples/android-app/proguard-rules.pro (hunk 2 above)
Comment (Tapchicoma) [reply samples/android-app/proguard-rules.pro]: Yes, this default android project structure.

File: settings.gradle (hunk 5 above)
Comment (Tapchicoma) [reply settings.gradle]: Changed in 471d810

File: settings.gradle (hunk 6 above)
Comment (Tapchicoma) [reply settings.gradle]: Changed in 471d810

File: samples/android-app/src/androidTest/java/by/egorr/myapplication/ExampleInstrumentedTest.kt (hunk 3 above)
Comment (Tapchicoma) [reply samples/android-app/src/androidTest/java/by/egorr/myapplication/ExampleInstrumentedTest.kt]: Fixed in 607f779

File: samples/android-app/src/main/java/org/jlleitschuh/gradle/ktlint/myapplication/ItemListActivity.kt (hunk 4 above)
Comment (Tapchicoma) [reply samples/android-app/src/main/java/org/jlleitschuh/gradle/ktlint/myapplication/ItemListActivity.kt]: Fixed in 607f779

**Comment by JLLeitschuh** [comment]
Woot! Gradle script Kotlin!!

I'll merge this when you think it's G2G

**Comment by Tapchicoma** [comment]
I think it is ok to merge.

**Pull request merged**